src/consensus/pbft/libbyz/Replica.h

```diff
@@ -250,6 +246,14 @@ class Replica : public Node, public IMessageReceiveBase
 #endif
   // Effects: Handle timeouts of corresponding timers.
 
+  // Playback methods
+  void playback_request(const pbft::Request& request, ccf::Store::Tx& tx);
```

Comment:
Comments before calls, with ///, for doxygen.

Reply:
The style in the rest of the PBFT code is

```
// Group description
function foo
// Foo description
function bar
// Bar description
```

I agree we should modernise these at some point, but for now its better to be consistent with the surrounding code.

Reply:
https://github.com/microsoft/CCF/issues/691